test/e2e/jobs/jobs.go

```diff
@@ -62,6 +62,20 @@ func DeleteTestInfra(subID, clusterName, location string) *types.Job {
 	return job
 }
 
+func InstallRetina(kubeConfigFilePath, chartPath string) *types.Job {
```

Comment:
Why do you need this? Can you refactor `InstallAndTestRetinaBasicMetrics`.

Reply:
I preferred not to change `InstallAndTestRetinaBasicMetrics` since this might be in use by someone else and the scope seems to be different. 

The plan for the new pipeline is to Create Infra, Install Retina, Scale Up and Run Tests, so my thinking is to create different jobs for each stage so each job could be reused somewhere else. So in the future instead of using `InstallAndTestRetinaBasicMetrics`, one could [re-]use `InstallRetina`, `Test Basic Metrics`, and perhaps add some other set of tests. Does that make sense?